### DISCUSSION TITLE
Domain verification error

### DISCUSSION THREAD

**Discussion by averytt** [WalletConnect/walletconnect-monorepo#2294] [Developer Support] [answered]
Hello,
Getting error on the dashboard: 

`Not found - DNS TXT verification record not found.`

We have confirmed the DNS record does exist on our domain for the last 14 days. 

Example:
`host -t TXT wallet-connect-dns-verification.thetatoken.org`

Returns (masked):
`wallet-connect-dns-verification.thetatoken.org descriptive text "5eebef4IS_THIS_SAFE_TO_SHARE_739d62a5f3"`

Thank you.

**Comment by finessevanes** [comment]
Your comment has been viewed and shared with our development team for review.

**Comment by bkrem** [comment, marked as answer]
Hi @averytt,

The record isn't being found because it only contains the value, not the key. For a valid entry the return should read:

```
wallet-connect-dns-verification.thetatoken.org descriptive text "wallet-connect-dns-verification=5eebef4IS_THIS_SAFE_TO_SHARE_739d62a5f3"
```

If you use the copy-to-clipboard functionality on Cloud for this field you'll have it in the expected format ready to go 👍
